### ISSUE TITLE
unittest fails if with Cplex

### ISSUE THREAD

**Issue by svigerske** [coin-or/OS#47]
Issue created by migration from Trac.

Original creator: @svigerske

Original creation time: 2011-12-29 12:35:00

Hi,

with the current trunk, I get
```
TEST 11: Cplex solver on p0033.osil

call the CPLEX Solver for p0033
CPX0000  Tried aggregator 2 times.
CPX0000  MIP Presolve eliminated 1 rows and 3 columns.
CPX0000  MIP Presolve modified 35 coefficients.
CPX0000  Aggregator did 2 substitutions.
CPX0000  Reduced MIP has 13 rows, 28 columns, and 91 nonzeros.
CPX0000  Reduced MIP has 28 binaries, 0 generals, 0 SOSs, and 0 indicators.
CPX0000  Probing time =    0.00 sec.
CPX0000  Tried aggregator 1 time.
CPX0000  Presolve time =    0.00 sec.
CPX0000  Probing time =    0.00 sec.
CPX0000  Clique table members: 4.
CPX0000  MIP emphasis: balance optimality and feasibility.
CPX0000  MIP search method: dynamic search.
CPX0000  Parallel mode: deterministic, using up to 4 threads.
CPX0000  Root relaxation solution time =    0.00 sec.
CPX0000
CPX0000          Nodes                                         Cuts/
CPX0000     Node  Left     Objective  IInf  Best Integer    Best Bound    ItCnt     Gap
CPX0000
CPX0000        0     0     2838.5467     6                   2838.5467       16
CPX0000  *     0+    0                         3095.0000     2838.5467       16    8.29%
CPX0000        0     0     2924.7588     7     3095.0000       Cuts: 6       22    5.50%
CPX0000        0     0     2941.4000     5     3095.0000      Cuts: 15       29    4.96%
CPX0000        0     0     2946.9927    13     3095.0000       Cuts: 5       45    4.78%
CPX0000        0     0     2949.0914    15     3095.0000       Cuts: 8       50    4.71%
CPX0000        0     0     2955.7720    16     3095.0000       Cuts: 3       52    4.50%
CPX0000        0     0     2962.7110    18     3095.0000       Cuts: 4       56    4.27%
CPX0000        0     0     2974.2469    17     3095.0000       Cuts: 5       61    3.90%
CPX0000        0     0     2985.7986    18     3095.0000       Cuts: 6       66    3.53%
CPX0000        0     0     3013.0000    17     3095.0000   ZeroHalf: 4       70    2.65%
CPX0000        0     0     3038.5000    17     3095.0000   ZeroHalf: 5       74    1.83%
CPX0000        0     0     3045.0588    17     3095.0000       Cuts: 2       76    1.61%
CPX0000  *     0+    0                         3089.0000     3045.0588       76    1.42%
CPX0000
CPX0000  Repeating presolve.
CPX0000  Tried aggregator 2 times.
CPX0000  MIP Presolve eliminated 1 rows and 8 columns.
CPX0000  MIP Presolve modified 17 coefficients.
CPX0000  Aggregator did 1 substitutions.
CPX0000  Reduced MIP has 11 rows, 19 columns, and 58 nonzeros.
CPX0000  Reduced MIP has 19 binaries, 0 generals, 0 SOSs, and 0 indicators.
CPX0000  Probing time =    0.00 sec.
CPX0000  Tried aggregator 1 time.
CPX0000  Represolve time =    0.00 sec.
CPX0000  Probing time =    0.00 sec.
CPX0000  Clique table members: 4.
CPX0000  MIP emphasis: balance optimality and feasibility.
CPX0000  MIP search method: dynamic search.
CPX0000  Parallel mode: deterministic, using up to 4 threads.
CPX0000  Root relaxation solution time =    0.00 sec.
CPX0000
CPX0000          Nodes                                         Cuts/
CPX0000     Node  Left     Objective  IInf  Best Integer    Best Bound    ItCnt     Gap
CPX0000
CPX0000  *     0+    0                         3089.0000     3045.0588       98    1.42%
CPX0000        0     0     3045.0588    13     3089.0000     3045.0588       98    1.42%
CPX0000        0     2     3045.0588    13     3089.0000     3045.0588       98    1.42%
CPX0000  Elapsed real time =
CPX0000  Elapsed real time CPX0000    0.02 sec. (tree size =  0.01 MB
CPX0000  Elapsed real time CPX0000    0.02 sec. (tree size =  0.01 MCPX0000  , solutions = 2
CPX0000
CPX0000  GUB cover cuts applied:  2
CPX0000  Cover cuts applied:  5
CPX0000  Zero-half cuts applied:  4
CPX0000  Gomory fractional cuts applied:  1
CPX0000
CPX0000  Root node processing (before b&c):
CPX0000    Real time             =    0.02
CPX0000  Parallel b&c, 4 threads:
CPX0000    Real time             =    0.00
CPX0000    Sync time (average)   =    0.00
CPX0000    Wait time (average)   =    0.00
CPX0000                            -------
CPX0000  Total (root+branch&cut) =    0.02 sec.
CPX0000  CPLEX Error  1262: No basis exists.
### ERROR: OsiCpxSolverInterface::getBasisStatus(): Unable to get base
```

It seems as if someone requests a basis after a MIP solve.

Stefan

**Comment by svigerske** [comment]
Comment by @svigerske created at 2013-09-01 12:25:04

Seems that this has been fixed with 98d9fea386, finally.

**Comment by svigerske** [comment]
Comment by @h-i-gassmann created at 2013-09-25 12:24:25

Resolution: fixed
